HW_1/task1.py

```diff
@@ -0,0 +1,13 @@
+# Найдите сумму цифр трехзначного числа.
+n = int(input('Введите трехзначное число: '))
+while n // 1000 > 0 :
+    if n//1000 > 0:
+        print('Число не трехзначное')
+        n = int(input('Введите трехзначное число: '))
+
+summa = 0
+while n > 0:
+    x = n % 10
+    summa = summa + x
```

Comment:
Если значение переменной используется единожды, то эту переменную использовать не нужно.
```suggestion
    summa += n % 10
```